packages/web-components/fast-foundation/src/checkbox/checkbox.template.ts

```diff
@@ -33,11 +39,16 @@ export function checkboxTemplate<T extends FASTCheckbox>(
             <label
                 part="label"
                 class="${x =>
-                    x.defaultSlottedNodes && x.defaultSlottedNodes.length
-                        ? "label"
-                        : "label label__hidden"}"
+                    ["label", x.defaultSlottedNodes?.length ? false : "label__hidden"]
+                        .filter(Boolean)
+                        .join(" ")}"
```

Comment:
Have we measured for any performance difference here? We should value performance and users over DX or elegance IMO.

Reply:
My main focus here was to remove the duplicated `label` class, which is a confusing repetition:

```ts
if (x.defaultSlottedNodes && x.defaultSlottedNodes.length) {
    return "label label__hidden";
} else {
    return "label";
}
```

This seems to have been done to avoid having an extra space character when `label__hidden` isn't present:

```ts
class="label ${x => x.defaultSlottedNodes && !x.defaultSlottedNodes.length && "label__hidden"}"
------------^
```

Though this should probably be addressed in a different isssue.